package.json

```diff
@@ -69,6 +69,7 @@
     "react-native-scale-bar": "^1.0.6",
     "react-native-screens": "^3.24.0",
     "react-native-svg": "^13.14.0",
+    "react-native-url-polyfill": "^2.0.0",
```

Comment:
had to introduce this because Hermes' `URL` implenetation does not have `pathname`, which is needed to parse the file URIs that are held in a draft photo object.